### PR TITLE
update:  example of applyStyle and applyFlex on the component positioning pages

### PR DESCRIPTION
**Pull request by luisoliveirazup**
**Description**

Update of the beagle code examples using applyStyle and applyFlex to use the new setFlex and setStyle implementation

### CODE REVIEW

File: content/en/resources/components-positioning/align-items.md [hunk 1]
```diff
@@ -78,41 +76,33 @@ Items are shifted to the cross axis start:
...
-                justifyContent = JustifyContent.SPACE_EVENLY,
-                alignItems = AlignItems.FLEX_START
-            )
-        )
+                createText(backgroundText = "#142850", text = "1")
+                      .setStyle {
+                                size = Size(
+                                    width = UnitValue.real(300),
+                                    height = UnitValue.real(100)
+                                )
```
Comment (matheusribeirozup) [inline content/en/resources/components-positioning/align-items.md:84]:
```suggestion
                                size = Size.box(width = 300, height = 100)
```

File: content/en/resources/components-positioning/align-items.md [hunk 2]
```diff
@@ -78,41 +76,33 @@ Items are shifted to the cross axis start:
...
+                      },
+                  createText(backgroundText = "#dd7631", text = "2")
+                      .setStyle {
+                          size = Size(
+                              width = UnitValue.real(200),
+                              height = UnitValue.real(100)
+                          )
```
Comment (matheusribeirozup) [inline content/en/resources/components-positioning/align-items.md:91]: Same suggestion: use `Size.box` instead.

File: content/en/resources/components-positioning/align-items.md (hunk 1 above)
Comment (luisoliveirazup) [reply content/en/resources/components-positioning/align-items.md]: change done!

File: content/en/resources/components-positioning/align-items.md (hunk 2 above)
Comment (luisoliveirazup) [reply content/en/resources/components-positioning/align-items.md]: change done!

**Review by matheusribeirozup — APPROVED** [review]

**Review by uziasferreirazup — APPROVED** [review]

**Pull request merged**